src/pages/Login-ResetPassword/login-resetpassword.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { FC, useState } from 'react';
+import Axios from 'axios';
+import Footer from '../../components/Footer/footer';
+import Navbar from '../../components/NavBar/navbar';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faUser } from '@fortawesome/free-solid-svg-icons';
+import Hackmercedlogo from '../../assets/images/hackmerced-logo.png';
+
+import './login-resetpassword.scss';
+
+const LoginResetPassword: FC = (): JSX.Element => {
+    const [form, setForm] = useState({ email: "" });
+
+    const handleInputChange = (
+        event: React.ChangeEvent<HTMLInputElement> | React.ChangeEvent<HTMLSelectElement>,
+    ): void => {
+        const { name, value } = event.target;
+        setForm({ ...form, [name]: value });
+    };
+
+    const handleSubmit = (event: React.ChangeEvent<HTMLFormElement>): void => {
+        event.preventDefault();
+        console.log(form);
+        Axios.post(`https://hackmerced-tomoe.herokuapp.com/v1/auth/login-resetpassword`, form).then(Response => console.log(Response));
+    }
+
+    return (
+        <>
+            <Navbar />
+            <main className="reset-page">
+                <div className="center-container">
+                    <img className="logo" src={Hackmercedlogo} alt="Logo" height={120} width={120} />
```

Comment:
```suggestion
                    <img className="logo" src={Hackmercedlogo} alt="Logo" />
```

The width and height for images should be in the SCSS files.